lib/versionist.js

```diff
@@ -248,9 +249,17 @@ exports.generateChangelog = (commits, options = {}, callback) => {
     if (error) {
       return callback(error);
     }
-    const parsableChangelogPath = '.versionbot/CHANGELOG.yml';
-    if (fs.existsSync(parsableChangelogPath)) {
-      const changelog = yaml.safeLoad(fs.readFileSync(parsableChangelogPath, 'utf8'));
+    const parsableChangelogFolder = '.versionbot';
+    const parsableChangelogName = 'CHANGELOG.yml';
+    const parsableChangelogPath = path.join(parsableChangelogFolder, parsableChangelogName);
+    if (fs.existsSync(parsableChangelogFolder)) {
+      let changelog = [];
+      try {
+        changelog = yaml.safeLoad(fs.readFileSync(parsableChangelogPath, 'utf8'));
+      } catch (e) {
+        // If no parsableChangelog exists, assume its empty
```

Comment:
Might well be worth catching on a `ENOENT` here, just to ensure it doesn't exist, rather then there being some other sort of fileop/error happening.